libvast/test/system/index.cpp

```diff
@@ -199,3 +218,16 @@ TEST(iterable bro conn log query result) {
 }
 
 FIXTURE_SCOPE_END()
+
+FIXTURE_SCOPE(meta_index_setup_test, synopsis_fixture)
+
+TEST(meta index factory) {
+  auto factory = get_synopsis_factory(sys);
```

Comment:
Any reason not to use `unbox(get_synopsis_factory(sys))`?

Reply:
Nope. Fixed.